Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -485,6 +489,35 @@ namespace gdjs {
       point[1] = position.y;
       return point;
     }
+    updateAntialiasing(): void {
+      if (this._object.getAntialiasing() != 'None') {
+        const antialiasingFilter =
+          this._antialiasingFilter ||
+          (this._antialiasingFilter = new PIXI.filters.FXAAFilter());
+        antialiasingFilter.enabled = true;
+        antialiasingFilter.multisample =
+          PIXI.MSAA_QUALITY[this._object.getAntialiasing().toUpperCase()] ||
+          PIXI.MSAA_QUALITY.LOW;
+
+        if (!this._graphics.filters) {
+          this._graphics.filters = [];
+        }
+        // does not apply filter if it already exists
```

Comment:
Comments are sentences. Use proper grammar.
```suggestion
        // Do not apply the filter if it is already present on the object.
```

Reply:
Sorry, I'll fix this